roster2/bundle/src/bundle/card_game_eu.properties

```diff
@@ -16,4 +16,7 @@ sound_enable = Gaitu soinu efektuak
 sound_disable = Desgaitu soinu efektuak
 user_image_alt = Asmatu behar duzun ikaslearen argazkia
 user_name_select_placeholder = Sartu ikaslearen izena...
-whats_the_user_name = Zein da ikaslearen izena?
\ No newline at end of file
+whats_the_user_name = Zein da ikaslearen izena?
+user_progress = {0} of {1} attempts were correct for this student.
```

Comment:
@pablosanroman sorry for bothering, could you provide these strings?

Reply:
Hello @mpellicer , @stetsche 

user_progress = {1} saiakeretatik {0} onak izan dira ikasle honentzat.
mark_learned = Ikasle hau ezagutzen dut, erronda honetan ez erakutsi niri.
no_name_found = Ez da izenik agertu bilaketa honetan

Reply:
Thank you very much! I've patched the file.